light/rpc/client.go

```diff
@@ -338,11 +341,32 @@ func (c *Client) BlockResults(height *int64) (*ctypes.ResultBlockResults, error)
 		return nil, err
 	}
 
-	// Verify block results.
+	// proto-encode BeginBlock events
```

Comment:
can't we use state.ABCIResponsesResultsHash func here?

Reply:
sure perhaps I've dipped too far into the direction of repeating yourself :)

Reply:
actually I realised that why I didn't do that is that the data structures don't really match up and it would be hard to create `ABCIResponses`